osquery/tables/system/BUCK

```diff
@@ -32,6 +32,7 @@ osquery_cxx_library(
             [
                 "posix/known_hosts.h",
                 "posix/shell_history.h",
+                "posix/ssh_keys.h",
```

Comment:
not sure if I need to modify buck files elsewhere; was struggling to get the buck build working.